pkg/internal/sqlprune/xwbtokens_test.go

```diff
@@ -1,255 +0,0 @@
-package sqlprune
-
-import (
-	"fmt"
-	"testing"
-)
-
-var globalTestingT *testing.T
-
-func setGlobalTestingT(t *testing.T) {
-	globalTestingT = t
-}
-
-func debugSQLParseOperationAndTable(tokenType int, data []byte) {
-	typeName, ok := xwbTokenTypeToName[tokenType]
-	if !ok {
-		typeName = fmt.Sprintf("Unknown (%d)", tokenType)
-	}
-	globalTestingT.Logf("Type %s=%s\n", typeName, string(data[:]))
-}
-
-// This is to be used for debugging unit tests, normally unused.
-var _ = debugSQLParseOperationAndTable
-
-var xwbTokenTypeToName = map[int]string{
-	57346: "LEX_ERROR",
```

Comment:
This was some left over debugging file, which was used to check token id to number mapping from xwbparser.